.test-infra/jenkins/job_LoadTests_GBK_Flink_Go.groovy

```diff
@@ -0,0 +1,220 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import CommonTestProperties
+import LoadTestsBuilder as loadTestsBuilder
+import PhraseTriggeringPostCommitBuilder
+import Flink
+import InfluxDBCredentialsHelper
+
+import static LoadTestsBuilder.DOCKER_CONTAINER_REGISTRY
+
+String now = new Date().format('MMddHHmmss', TimeZone.getTimeZone('UTC'))
+
+def batchScenarios = {
+  [
+    [
+      title          : 'Group By Key Go Load test: 2GB of 10B records',
+      test           : 'group_by_key',
+      runner         : CommonTestProperties.Runner.FLINK,
+      influx_namespace: 'flink',
+      pipelineOptions: [
+        job_name           : "load-tests-go-flink-batch-gbk-1-${now}",
+        influx_measurement : 'go_batch_gbk_1',
+        input_options      : '\'{' +
+        '"num_records": 200000000,' +
+        '"key_size": 1,' +
+        '"value_size": 9}\'',
+        iterations         : 1,
+        fanout             : 1,
+        parallelism        : 5,
+        endpoint           : 'localhost:8099',
+        environment_type   : 'DOCKER',
+        environment_config : "${DOCKER_CONTAINER_REGISTRY}/beam_go_sdk:latest",
+      ]
+    ],
+    [
+      title          : 'Group By Key Go Load test: 2GB of 100B records',
+      test           : 'group_by_key',
+      runner         : CommonTestProperties.Runner.FLINK,
+      influx_namespace: 'flink',
+      pipelineOptions: [
+        job_name           : "load-tests-go-flink-batch-gbk-1-${now}",
```

Comment:
It seems you forgot about changing the "1" to "2" and so on. It doesn't really matter in case of `job_name`, but it does matter in case of `influx_measurement`: all results would be put into the first chart, leaving other charts empty.